gravitee-apim-portal-webui-next/src/app/api-details/subscribe-to-api/subscribe-to-api-choose-plan/subscribe-to-api-choose-plan.component.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, Input } from '@angular/core';
+import { FormControl, FormGroup, FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { MatButton, MatButtonModule } from '@angular/material/button';
+import { MatCard, MatCardActions, MatCardContent, MatCardHeader } from '@angular/material/card';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatIcon } from '@angular/material/icon';
+import { MatInputModule } from '@angular/material/input';
+import { MatRadioModule } from '@angular/material/radio';
+import { MatStepperModule } from '@angular/material/stepper';
+import { MatTabLink, MatTabNav, MatTabNavPanel } from '@angular/material/tabs';
+import { RouterLinkActive, RouterOutlet } from '@angular/router';
+
+import { BannerComponent } from '../../../../components/banner/banner.component';
+import { BreadcrumbNavigationComponent } from '../../../../components/breadcrumb-navigation/breadcrumb-navigation.component';
+import { PlanCardComponent } from '../../../../components/subscribe/plan-card/plan-card.component';
+import { SubscribeToApiCheckoutComponent } from '../subscribe-to-api-checkout/subscribe-to-api-checkout.component';
+import { SubscribePlan } from '../subscribe-to-api.component';
+
+@Component({
+  selector: 'app-subscribe-to-api-choose-plan',
+  standalone: true,
+  imports: [
+    MatCardContent,
+    PlanCardComponent,
+    MatCardActions,
+    MatCardHeader,
+    BannerComponent,
+    BreadcrumbNavigationComponent,
+    MatButton,
+    MatCard,
```

Comment:
what about using the module to avoid 15O imports line ?
(optional juste thought )

Reply:
Good idea, just created the module for subscription-to-api